cli_reference/developer_cli_odo/odo-getting-started.adoc

```diff
@@ -0,0 +1,59 @@
+:_content-type: ASSEMBLY
+[id='odo-getting-started']
+= Getting started with `{odo-title}`
+include::_attributes/common-attributes.adoc[]
+:context: odo-getting-started
+
+toc::[]
+
+Learn how to use odo to create a "Hello World" application, based on one of the following frameworks:
+
+* Node.js
+* .NET
+* Java (Spring Boot)
+* Go
+
+== Prerequisites
+
+* You have the xref:installing-odo.adoc#installing-odo[odo binary] installed.
+* You have access to an {product-title} cluster.
+
+== Generating sample code
+
+This tutorial uses a sample project for each framework, but you can choose other examples from the list provided by the `odo registry` command. Alternatively, you can use an exisiting project or you can run `odo init` and select a starter project.
+
+include::modules/odo-getting-started-code-nodejs.adoc[leveloffset=+2]
+include::modules/odo-getting-started-code-dot-net.adoc[leveloffset=+2]
+include::modules/odo-getting-started-code-java.adoc[leveloffset=+2]
+include::modules/odo-getting-started-code-go.adoc[leveloffset=+2]
+
+include::modules/odo-getting-started-cluster-login-openshift.adoc[leveloffset=+1]
+include::modules/odo-getting-started-create-project.adoc[leveloffset=+1]
+
+
+== Initializing your application (odo init)
+
+Initialize your application by creating a `devfile.yaml` that will be deployed. `odo` handles this automatically with the `odo init` command by autodetecting your source code and downloading the appropriate Devfile.
```

Comment:
```suggestion
Initialize your application by creating a `devfile.yaml`. When you run the `odo init` command, `odo` autodetects your source code, downloads the appropriate devfile, and deploys the application.
```

Reply:
- "will be deployed" - Avoid using the future tense and passive voice. Use the present tense and active voice instead.